backend/ops_api/ops/environment/default_settings.py

```diff
@@ -1,17 +1,27 @@
-from datetime import timedelta
 import os
+from datetime import timedelta
 
 DEBUG = False  # make sure DEBUG is off unless enabled explicitly otherwise
-SQLALCHEMY_DATABASE_URI = "sqlite://"
+
+# SQLALCHEMY_DATABASE_URI = "postgresql+psycopg2://postgres:local_password@localhost:5432/postgres"
 SQLALCHEMY_TRACK_MODIFICATIONS = False
+SQLALCHEMY_ECHO = False
+
 FLASK_PORT = 8080
+
+JWT_PRIVATE_KEY = os.getenv("JWT_PRIVATE_KEY")
+JWT_PUBLIC_KEY_PATH = "./static/public.pem"
+JWT_ALGORITHM = "RS256"
+JWT_DECODE_ALGORITHMS = "RS256"
+JWT_TOKEN_LOCATION = "headers"  # noqa: S105 "Not a secret"
+JWT_ACCESS_TOKEN_EXPIRES = timedelta(minutes=15)  # FedRAMP AC-12 Control
+JWT_REFRESH_TOKEN_EXPIRES = timedelta(hours=12)
+
 AUTHLIB_OAUTH_CLIENTS = {
     "logingov": {
         "server_metadata_url": "https://idp.int.identitysandbox.gov/.well-known/openid-configuration",
+        "user_info_url": "https://idp.int.identitysandbox.gov/api/openid_connect/userinfo",
         "client_id": "urn:gov:gsa:openidconnect.profiles:sp:sso:hhs_acf:opre_ops",
-        "client_kwargs": {"scope": "openid"},
-    }
+        "client_kwargs": {"scope": "openid email"},
```

Comment:
I think we will want the first and last name from `profile`

Reply:
We ultimately will, and that locks us into using AAL2 as well, maybe flush that out more when we fully lockdown our "User" needs.